### ISSUE TITLE
BUG: The installer URL from Kinto.sh is installing xkeysnail v0.4.0 with missing patches for held keys

### ISSUE THREAD

**Issue by RedBearAK** [rbreaves/kinto#618]
@rbreaves 

Maybe I used confusing phrasing on my previous bug report. Attempting to clarify: 

Doing a fresh install on a new system using the copy/paste URL found at https://kinto.sh ends up installing a version of `xkeysnail` which is missing your patches for held keys, and reports that its version is v0.4.0. So it is **not** installing the patched v0.3.0 version that you expect it to install. 

The URL that I copy from the https://kinto.sh website and paste in a terminal: 

```
/bin/bash -c "$(wget -qO- https://raw.githubusercontent.com/rbreaves/kinto/HEAD/install/linux.sh || curl -fsSL https://raw.githubusercontent.com/rbreaves/kinto/HEAD/install/linux.sh)"
```

There seems to be a mix-up with the branch that the URL installer is installing from. Please look into it. Try it in a VM. You should see the installed version of `xkeysnail` will be v0.4.0 and will **not** have any of your patches for held keys. 

Anyone who attempts to use the URL from your website should end up seeing the same thing I'm seeing, and their Alt+Tab and Alt+Grave will not work correctly. 

Probably also happening when people manually download your master branch and do a manual install with `setup.py`. Since it will have the same installer script with the same command that will clone your branch of `xkeysnail` and wind up installing v0.4.0 with no patches. 

I believe it has been this way for anywhere from two weeks (just before I tested CutefishOS) to three weeks, since my install on Ubuntu 21.04 on the 5th worked just fine and the installed `xkeysnail` reports as v0.3.0 and has your patches.

**Comment by RedBearAK** [comment]
@rbreaves 

Reconfirming: 

Just threw Ubuntu 21.04 into a clean VM and immediately installed Kinto from the website URL. It installs v0.4.0 and Alt+Tab is broken. 

As opposed to the Ubuntu 21.04 hosting the Ubuntu 21.04 guest VM in Boxes, which was installed exactly the same way (from the website URL), has v0.3.0 and Alt+Tab/Alt+Grave has always worked fine. 

Just for good measure I reverted the VM to a clean state and then did the old `git clone https://github.com/rbreaves/kinto` that I used to do before the website URL existed, and ran `setup.py` manually. As expected, since it's probably the same set of files that get downloaded either way, it installed v0.4.0 and Alt+Tab was broken. 

If you were wanting the installer to pull from the "kinto" branch of `rbreaves/xkeysnail` to avoid this, you'll need to modify the installer to pull directly from that branch or tree or whatever. It seems to be pulling from "master" which is v0.4.0 and no patches. 

I think your advanced and frequent usage of `git` features like checking out a different branch has led you to an erroneous conclusion about why I was seeing this. I had git working with VSCode once upon a time, but that install of Ubuntu blew itself up (for unknown reasons) and I had to do a fresh install of Ubuntu on a different disk. Other than the fact that the Kinto installer put `git` on my system, I don't use it at all. 

On that note, I suggest avoiding installing Ubuntu with ZFS-on-root until they put a lot more polish on their ZFS support. Trying to recover files from such a disk is a huge PitA the way they've set it up so far. It is immensely easier to work with any other LVM volume, encrypted or not.

**Comment by rbreaves** [comment]
Closing this - re-opened #616 till confirmation that the issue is solved. I have resolved the issue on both ends now - the xkeysnail fork and the installer script to make sure the kinto branch is used going forward - but since older releases default to master (and shouldn't) I will leave master back on 0.3.0. Latest will go into a branch named latest, master & kinto will be frozen to 0.3.0 as far as I am concerned.

Future releases and modifications to xkeysnail will result in newer branches, but the existing ones will stay where they are at with the exception to the new "latest" branch.
